app/components/FileSearchBar.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
```

Comment:
Ah shit could you please use 4 space indentation like the rest of the files. VSCode can do that automatically for you.